common/buildcraft/api/bpt/BlueprintAPI.java

```diff
@@ -12,7 +12,7 @@
 import net.minecraft.util.ResourceLocation;
 import net.minecraft.util.StringUtils;
 
-import buildcraft.api.IUniqueReader;
+//import buildcraft.api.IUniqueReader;
```

Comment:
i would advise agains commenting out imports, it's easy to have your IDE auto-re-import but the comment will stay there being useless